stencil-library/src/components/payments-list/payments-list.tsx

```diff
@@ -73,22 +73,31 @@ export class PaymentsList {
 
     this.loading = true;
 
-    const api = Api(this.authToken, config.proxyApiOrigin);
-    const endpoint = `account/${this.accountId}/payments`;
-    const response: IApiResponseCollection<IPayment[]> = await api.get(endpoint, this.params);
+    try {
+      const api = Api(this.authToken, config.proxyApiOrigin);
+      const endpoint = `account/${this.accountId}/payments`;
 
-    if (!response.error) {
-      this.paging = {
-        ...this.paging,
-        ...response.page_info
-      }
+      const response: IApiResponseCollection<Payment[]> = await api.get(endpoint, this.params);
```

Comment:
Similar to payment-details - can you please re-import `IPayment` and set the type for this response on line 80 to:

`response: IApiResponseCollection<IPayment[]>`